spec/requests/memory_spec.rb

```diff
@@ -19,5 +18,25 @@ def count_instances_of(klass)
       expect(count_instances_of klass).to be <= count
     end
   end
+end
 
+describe "Memory Leak", :type => :request, :if => RUBY_ENGINE == 'jruby' do
+  before do
+    require "java"
+    java_import 'java.lang.management.ManagementFactory'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.